django/bosstiles/views.py

```diff
@@ -16,9 +16,12 @@
 from rest_framework.permissions import IsAuthenticated
 from django.conf import settings
 
+from boss.utils import BossUtils
 from bosscore.request import BossRequest
 from bosscore.error import BossError, BossHTTPError, ErrorCodes
 
+from bossutils.logger import BossLogger
```

Comment:
Is this used?

Reply:
No it's not, sorry, I used it during testing